src/content-script/listeners/player.ts

```diff
@@ -47,40 +49,68 @@ function findVideoPlayer(): HTMLVideoElement | null {
 //------------------------------------------------------------------------------
 
 function addVideoListeners(player: HTMLVideoElement) {
-  // TODO: hostRequired should default to true once that's implemented
-  function sendVideoEvent(socketEvent: SocketEvents, hostRequired = false) {
+  // TODO: Once implemented, include communism/facism check
+  function sendVideoEvent(socketEvent: VideoSocketEvents, hostRequired = true) {
     return (event: any) => {
       const state = store.getState();
-      const { isHost } = state.party;
-      const { id: partyId } = state.party;
-      const { currentTime, duration, playbackRate } = event.target;
+      const { isHost, hash } = state.party;
 
-      const payload = { partyId, currentTime, duration, playbackRate };
+      // if party not created, don't send anything
+      if (!hash) return;
 
+      // construct payload
+      const { paused, currentTime, duration, playbackRate } = event.target;
+      const eventData: VideoEventData = { eventType: socketEvent, isHost, paused, currentTime, playbackRate, duration };
```

Comment:
Let's not send `isHost` for now. If users are receiving player events, they know it's from the host because we're only sending in that case. If we decide to revisit this after MVP functionality is there, then this would be a good addition.